_scss/_night-mode.scss

```diff
@@ -91,10 +91,21 @@ body.night {
         color: #f3f3f3;
         background-color: #0a121b !important;
         border: 1px solid $black;
+
+        @media print {
+            color: #444;
+            background-color: #f5f8fa !important;
+            border-width: 0;
+        }
```

Comment:
Instead of overriding these, perhaps it would be possible to make the "night-mode" style conditional (i.e., don't apply dark mode in "print")

Reply:
I don't think this will be easy to achieve. ... The original author didn't use a media query to select styles for dark mode. He/she used a class (`body.night`) and applied it conditionally to the `<body>` element using JavaScript. So, I cannot alter a media query to omit the `print` media.